tests/unittests/torch_dataset_tests.py

```
@@ -9,11 +9,33 @@
 import unittest
 
 import fiftyone as fo
-from fiftyone.utils.torch import FiftyOneTorchDataset
+from fiftyone.utils.torch import FiftyOneTorchDataset, GetItem
 
 
-def id_get_item(sample):
-    return sample["id"]
+class IdentityGetItem(GetItem):
+    def __init__(self, required_keys=["filepath"], **kwargs):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)